app/konnect/getting-started/migration.md

```diff
@@ -0,0 +1,318 @@
+---
+title: Migrating from Kong Gateway On-Premises to Kong Konnect
```

Comment:
From the title I'm expecting a step-by-step guide on how to migrate. The steps written feel more abstract, and I understand that everyones case is different but if we want to use illustrative steps maybe we can reorder them so that they express a recommended order of operations? 

Like: 

1. Export config
2. Modify Config 
3. Import config
4. Setup RBAC 
5. Migrate Data planes 
Then you have extra information that may not apply to everyone like: 
Handling plugins